src/app/components/App.tsx

```diff
@@ -286,174 +288,176 @@ const App = () => {
-                  <SingleColumnLayout>
-                    <BlastForm />
-                  </SingleColumnLayout>
-                )}
-              />
-              <Route
-                path={LocationToPath[Location.AlignResult]}
-                component={AlignResult}
-              />
-              <Route
-                path={LocationToPath[Location.Align]}
-                render={() => (
-                  <SingleColumnLayout>
-                    <AlignForm />
-                  </SingleColumnLayout>
-                )}
-              />
-              <Route
-                path={LocationToPath[Location.PeptideSearchResult]}
-                component={PeptideSearchResult}
-              />
-              <Route
-                path={LocationToPath[Location.PeptideSearch]}
-                render={() => (
-                  <SingleColumnLayout>
-                    <PeptideSearchForm />
-                  </SingleColumnLayout>
-                )}
-              />
-              <Route
-                path={LocationToPath[Location.IDMappingResult]}
-                component={IDMappingResult}
-              />
-              <Route
-                path={LocationToPath[Location.IDMapping]}
-                render={() => (
-                  <SingleColumnLayout>
-                    <IDMappingForm />
-                  </SingleColumnLayout>
-                )}
-              />
-              <Route
-                path={LocationToPath[Location.Dashboard]}
-                render={() => (
-                  <SingleColumnLayout>
-                    <Dashboard />
-                  </SingleColumnLayout>
-                )}
-              />
-              <Route
-                path={LocationToPath[Location.Basket]}
-                component={BasketFullView}
-              />
-              <Route
-                path={LocationToPath[Location.HelpEntry]}
-                component={HelpEntryPage}
-              />
-              <Route
-                path={LocationToPath[Location.HelpResults]}
-                component={ResultsOrLanding(HelpResults, HelpLandingPage)}
-              />
-              <Route
-                path={LocationToPath[Location.Contact]}
-                component={() => {
-                  // Temporary redirect to current website
-                  window.location.href = 'https://www.uniprot.org/contact';
-                  return null;
-                }}
-              />
-              {/* Catch-all handler -> Redirect or not found use ResourceNotFoundPage */}
-              <Route
-                path="*"
-                render={() => (
-                  <SingleColumnLayout>
-                    <ResourceNotFoundPage />
-                  </SingleColumnLayout>
-                )}
-              />
-            </Switch>
-          </Suspense>
-        </BaseLayout>
-        <ErrorBoundary fallback={null}>
-          <GDPR />
-        </ErrorBoundary>
+        <DBMapsProvider>
```

Comment:
This file is already quite busy with all the routes, and we might end up adding some other top-level contexts eventually.
I would create a file exporting a GlobalContexts component, containing the ReduxProvider in index.tsx, and FranklinSite, Router, and DBMapsProvider. and make the index.tsx look like
```
ReactDOM.render(
  <GlobalContexts store={store} history={history}>
    <App />
  </GlobalContexts>,
  document.getElementById('root')
);
```
So that any further global context that we might add in the future, we would put with the rest of them.

Reply:
(Sorry, the GitHub preview for that comment doesn't really look like what I was expecting, it's meant to cover these lines:
```
<FranklinSite>
  <Router history={history}>
    <DBMapsProvider>
```
)

Reply:
That's a nice idea - I'll do that now